mockups/EX425/Dockerfile

```diff
@@ -54,10 +55,10 @@ RUN set -ex \
     && pip3 install wheel \
     && pip3 install -r /app/requirements.txt
 
-EXPOSE 5000
+EXPOSE 443
 ENV MOCKUPFOLDER="EX425"
-ENV AUTH_CONFIG=""
-ENV PORT=5000
+ENV AUTH_CONFIG="root:initial0:Administrator"
```

Comment:
We should not have credentials anywhere in github.  As this is an open source repo, this should probably be removed even though the PR was closed.